### PR TITLE
Fix redis healthcheck

### PR DESCRIPTION
**Pull request by roborourke**
This is an effort to make the healthcheck backwards compatible in case wp-redis changes the stats method behaviour.

### CODE REVIEW

File: inc/healthcheck/namespace.php [hunk 1]
```diff
@@ -60,7 +60,7 @@ function output_page( array $checks ) {
 	if ( ! empty( $_SERVER['HTTP_ACCEPT'] ) && $_SERVER['HTTP_ACCEPT'] === 'application/json' ) {
 		$format = 'json';
 	}
-	if ( $_GET['_accept'] ?? '' === 'json' ) {
+	if ( isset( $_GET['_accept'] ) && $_GET['_accept'] === 'json' ) {
```
Comment (rmccue) [inline inc/healthcheck/namespace.php:63]: Why is this changed?

Comment (roborourke) [reply inc/healthcheck/namespace.php:63]: Was getting some linter errors but I can change it back

**Review by jazzsequence — APPROVED** [review]
